EDMarketConnector.py

```diff
@@ -127,8 +127,18 @@
                         nargs='*'
                         )
 
+    parser.add_argument(
+        '--capi-pretend-down',
+        help='Force to raise ServerError on any CAPI query',
+        action='store_true')
```

Comment:
```suggestion
        action='store_true'
    )
```
Please use hanging braces if you want to multiline like this